src/theme-provider.js

```diff
@@ -0,0 +1,75 @@
+import React, {PropTypes, Component} from 'react'
+import brcast from 'brcast'
+
+export const CHANNEL = '__glamorous__'
+
+/**
+ * This is a component which will provide a theme to the entire tree
+ * via context and event listener (because pure components block context updates)
```

Comment:
The build is failing because this line is too long. You might run `npm start validate` before you commit to check that :+1: